components/signup.jsx

```diff
@@ -1,29 +1,11 @@
 import React from 'react';
-import {IntlMixin, FormattedHTMLMessage} from 'react-intl';
+import IntlMixin from 'react-intl';
+import Email from '../components/email-input.jsx';
```

Comment:
I like Email too, but right now it seems like we lost all the `<input>` at the bottom there which makes all these `input` that you are adding not obvious until you look at the top here? I don't mind just the `Email` but I just want us to stick with something that will make it clear for other people to look and understand right away what's going on here, so if you think it is already very clear then leave it as is.